package.json

```diff
@@ -36,6 +36,13 @@
   },
   "homepage": "https://github.com/1-Platform/one-platform#readme",
   "scripts": {
+    "clean:reports": "rm -R -f e2e/reports && mkdir e2e/reports",
+    "pretest": "npm run clean:reports ",
```

Comment:
This wont be executed since there is no `test` script spacified right?

Reply:
These are to clear the old reports if you want to, as they won't get overridden.